package.json

```diff
@@ -33,7 +33,7 @@
   },
   "devDependencies": {
     "asar": "^2.0.1",
-    "electron-builder": "^21.2.0",
+    "electron-builder": "^22.3.2",
     "electron-builder-squirrel-windows": "^21.2.0",
```

Comment:
I tend to also upgrade this one as well to the same version, as otherwise it screams about mismatching peer deps or something similar.

Reply:
also an excellent point